.ci.yaml

```diff
@@ -331,6 +351,28 @@ targets:
       xcode: 14a5294e # xcode 14.0 beta 5
     timeout: 75
 
+  - name: Mac Host clang-tidy
+    recipe: engine/engine_lint
+    bringup: true
+    properties:
+      add_recipes_cq: "true"
+      jazzy_version: "0.14.1"
+      xcode: 14a5294e # xcode 14.0 beta 5
+      lint_host: "true"
+      lint_ios: "false"
```

Comment:
I guess there was a recipe change to read these?

Reply:
Yes. I added a link to it in the PR description.